### ISSUE TITLE
interactive tutorial not working 

### ISSUE THREAD

**Issue by rupeshpadhye** [appbaseio/dashboard#31]
while trying out interactive tutorial I am getting "An app with the same name exists" for all the unique inputs .

**Comment by metagrover** [comment]
I just checked. It works for me. Currently, the way it works - your app-name should be unique across the entire appbase ecosystem. I suggest you to try alpha-numeric app-names.

**Comment by rupeshpadhye** [comment]
I tried multiple unique names few names as below 

rpmum124india,rpmum124india,ro@earheloo7rjk

sharing screen shot 

![image](https://user-images.githubusercontent.com/10848048/36605433-b3bfff4e-18e6-11e8-97b1-9262560948ae.png)


![image](https://user-images.githubusercontent.com/10848048/36605473-d3bc1634-18e6-11e8-8059-316079ef2f89.png)

**Comment by siddharthlatest** [comment]
@rupeshpadhye I am not able to replicate the issue. I tried the app names that you mention and am able to create them. Can you try again?

What OS/browser/version are you using? Do you see any errors in the developer console?

**Comment by rupeshpadhye** [comment]
@siddharthlatest  I checked console while creating app name helllo34342ks38mod124krp7   and 
status is 401 unauthorized 
> https://accapi.appbase.io/app/helllo34342ks38mod124krp7

`{"message":"Not Logged In, No Token"}`

 I haven't created account yet and just trying the interactive tutorial . Do i need to create account to test interactive tutorial .

**Comment by siddharthlatest** [comment]
Yep, you will need to create an account before you try the tutorial. We should have shown a login prompt. How were you able to access it without logging in?
